plio/views.py

```diff
@@ -81,10 +122,15 @@ def perform_update(self, serializer):
     @action(detail=False, permission_classes=[IsAuthenticated])
     def list_uuid(self, request):
         """Retrieves a list of UUIDs for all the plios"""
-        queryset = self.get_queryset()
-        if not queryset or not queryset.exists():
-            return Response([])
-        return Response(list(queryset.values_list("uuid", flat=True)))
+        queryset = self.filter_queryset(self.get_queryset())
```

Comment:
shouldn't the check for `if not queryset or not queryset.exists()` happen before `filter_queryset`? 
Has this been tested for ids with no plios to list?